EventsExpress/Mapping/EventScheduleAutoMapper.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Linq;
+using AutoMapper;
+using EventsExpress.Core.DTOs;
+using EventsExpress.Core.Extensions;
+using EventsExpress.Db.Entities;
+using EventsExpress.DTO;
+
+namespace EventsExpress.Mapping
+{
+    public class EventScheduleAutoMapper : Profile
+    {
+        public EventScheduleAutoMapper()
+        {
+            CreateMap<EventSchedule, EventScheduleDTO>()
+           .ForMember(dest => dest.Id, opts => opts.MapFrom(src => src.Id))
+           .ForMember(dest => dest.Frequency, opts => opts.MapFrom(src => src.Frequency))
+           .ForMember(dest => dest.Periodicity, opts => opts.MapFrom(src => src.Periodicity))
+           .ForMember(dest => dest.LastRun, opts => opts.MapFrom(src => src.LastRun))
+           .ForMember(dest => dest.NextRun, opts => opts.MapFrom(src => src.NextRun))
+           .ForMember(dest => dest.IsActive, opts => opts.MapFrom(src => src.IsActive))
```

Comment:
You can skip this, as the properties with the same names will be mapped by default